templates/taxonomy-topic.php

```diff
@@ -55,13 +56,16 @@
 				<?php
 					echo $related_faq_html;
 				}
-
 			?>
+					<a href="<?php echo site_url(); ?>/faq/" class="<?php UCF_FAQ_Config::add_athena_attr( 'btn btn-primary mt-4' ); ?>">View all FAQs</a>
```

Comment:
I'd suggest making this href configurable as a plugin setting (but default to `site_url()` . /faq/)